docs/build/build-with-ai/mcp.md

```diff
@@ -0,0 +1,105 @@
+---
+title: MCP Server
+description: Learn how to get started with MCP servers and implement the Model Context Protocol
+---
+
+# Model Context Protocol (MCP)
+
+[The Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction) is an open protocol that standardizes how applications provide context to Large Language Models (LLMs). It was developed to solve the challenge of consistently and efficiently connecting AI models with various data sources and tools.
+
+## Why MCP?
+
+MCP helps you build agents and complex workflows on top of LLMs by providing:
+
+- A growing list of pre-built integrations that your LLM can directly plug into
+- The flexibility to switch between LLM providers and vendors
+- Best practices for securing your data within your infrastructure
+
+## Core Architecture
+
+MCP follows a client-server architecture where a host application can connect to multiple servers:
+
+### Components
+
+- **MCP Hosts**: Programs like Claude Desktop, IDEs, or AI tools that want to access data through MCP
+- **MCP Clients**: Protocol clients that maintain 1:1 connections with servers
+- **MCP Servers**: Lightweight programs that each expose specific capabilities through the standardized Model Context Protocol
+- **Local Data Sources**: Your computer's files, databases, and services that MCP servers can securely access
+- **Remote Services**: External systems available over the internet (e.g., through APIs) that MCP servers can connect to
+
+## Getting Started
```

Comment:
I find the subsections here not very practical. It reads like an outline of things to do, without saying how.